package.json

```diff
@@ -18,7 +18,7 @@
     "karma-spec-reporter": "0.0.13",
     "karma-osx-reporter": "*",
     "mocha": "~1.15.1",
-    "phantomjs": "1.9.7-1",
```

Comment:
Unfortunately not, this is back to how it was before. We tried to resolve it last night by specifying an exact version of PhantomJs but we still couldn't get the test to run. That's why we're removing this test (setting to WIP)!